Source/Tools/PR.viancfg

```diff
@@ -487,8 +487,13 @@
 		<MaxFailures>5</MaxFailures>
 		<BasePath>"LabVIEW"</BasePath>
 		<RelativePath>"project\\_VI Analyzer\\_tests\\Complexity Metrics\\Depth of Nesting of Structures.llb"</RelativePath>
-		<Selected>FALSE</Selected>
+		<Selected>TRUE</Selected>
 		<Controls>
+			<Control>
+				<Name>"Maximum Nesting Depth"</Name>
+				<Type>"Int"</Type>
+				<Value>11</Value>
```

Comment:
`Initialize Sessions Impl.vi` is a beast of a VI. It probably should be simplified. Other than `Initialize Sessions Impl.vi`, which VI has the next highest level of nesting? 

Can you remove IPE structures from the check? I don't consider IPE structures true nesting like I would a case structure or loop (although I'm sure someone would debate me on this).

Reply:
I agree that if `Initialize Sessions Impl.vi` fails the check, then the rule is probably doing what we want it to do. We've acknowledged on several occasions that the VI should be refactored to make it more readable/maintainable. We just haven't done the work. I'd be curious to know what other VIs were flagged to get a better sense on the calibration of the rule. If `Initialize Sessions Impl.vi` doesn't trigger a violation or require an exemption, then the rule is probably worthless and not worth enabling.

Reply:
These are the four VIs that fail the check when it's configured with its default depth of 6:

```
Close Sessions.vi (C:\github\measurementlink-labview\Source\Runtime\MeasurementLink Session Management V1\Session Reservation\Close Sessions.vi)
Initialize Sessions Impl.vi (C:\github\measurementlink-labview\Source\Runtime\MeasurementLink Session Management V1\Session Reservation\Initialize Sessions Impl.vi)
Remove All Sessions.vi (C:\github\measurementlink-labview\Source\Runtime\MeasurementLink Session Management V1\Connection Map\Remove All Sessions.vi)
Service Location FGV Impl.vi (C:\github\measurementlink-labview\Source\Runtime\MeasurementLink Discovery V1\Service Location FGV Impl.vi)
```

Reply:
There does not appear to be any configurable means to filter particular structures such as IPE from the check.
Here are the maximum nested depths of the offending VIs:
- **Initialize Sessions Impl.vi** = 11 levels
- **Remove All Sessions.vi** = 9 levels
- **Service Location FGV Impl.vi** = 8 levels
- **Close Sessions.vi** = 7 levels

Reply:
`Initialize Sessions Impl.vi` is bad and should be refactored. I'm indifferent on the other 3. `Remove All Sessions.vi` is just a bunch of for loops and IPE structures. It is surprisingly simple and no one is going to be confused by the 9 layers of nesting. All 3 could be simplified if necessary. Because there are only 4 hits, I'm inclined to believe that this check is reasonable and won't be overly burdensome on developers.

Reply:
fixed in 1-11-24 update: Left in the check with tolerance of 8; made exceptions for the two cases that exceed it.